templates/note/message_view.html

```diff
@@ -32,6 +32,17 @@
 
     <div class="formatted-content">$:{MARKDOWN(query['content'])}</div>
 
+    $if myself['is_mod']:
+      <form name="copynotetostaffnotes" action="/modcontrol/copynotetostaffnotes" method="post" class="form clear">
+        <button type="submit" class="button positive" style="float: right;">Copy Note to Staff Notes</button>
+        <input type="hidden" name="notecontent" value="$:{MARKDOWN(query['content'])}" />
```

Comment:
I think this should use the note id instead of submitting all the information through a form. If that weren’t the case, though, you still wouldn’t want to use `$:` here, or pass the content through the Markdown filter for that matter.